templates/base.html

```diff
@@ -63,6 +63,7 @@
     <meta property="og:title" content="{{ title|safe }}">
     <meta property="og:url" content="{{app.site.url}}{{ request.path }}">
     <meta property="og:language" content="fr_FR">
+    <meta property="og:image" content="https://{{ meta_image }}">
```

Comment:
Je suis pas sur à quel point l'outil scrapper de Facebook, le gère. 

Pour l'attribut "og:image" et "og:image:url", sur la documentation, y'a écrit, "og:image - An image URL which should represent your object within the graph." mais comme il donne pas la définition d'une URL . Dans leurs exemples de présentation, il précise les http.

Pour l'attribut "og:image:secure_url", il mette clairement "An alternate url to use if the webpage requires HTTPS." donc ça me parait logique de préciser.